### PR TITLE
[README] Add citation section

### LINKED ISSUE #7255: Preferred way to cite/acknowledge Kratos in papers?
**Description**
Is there any article to cite, or a text for the acknowledgments (citing grants, institutions, etc.) in the publications?
Any info that must go?

**Additional info**
I don't find this information in the wiki, maybe we can add a "Citing Kratos" article.

### PR DESCRIPTION
**Pull request by riccardotosi**
**Description**
This PR adds a new section to the README.

Please mark the PR with appropriate tags: 
- Documentation.

**Changelog**
Adding three references to the readme for citing Kratos.

Please note that in the [Zenodo web-page](https://zenodo.org/record/3870709#.XxW-f-exVPY), it is written that

> Cite all versions? You can cite all versions by using the DOI 10.5281/zenodo.3234644. This DOI represents all versions, and will always resolve to the latest one.

I have copy-pasted the references as the journals and Zenodo suggest, and some name are not being written in the reference.

**Additional info**
None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,3 +90,9 @@ In Kratos Core:
 In applications
 - [Trilinos](https://trilinos.org/) for MPI linear algebra and solvers used in trilinos application
 - [METIS](http://glaros.dtc.umn.edu/gkhome/views/metis) for partitioning in metis application
+
+# How to cite Kratos?
+Please, use the following references when citing Kratos in your work.
+- Dadvand, P., Rossi, R. & Oñate, E. An Object-oriented Environment for Developing Finite Element Codes for Multi-disciplinary Applications. Arch Computat Methods Eng 17, 253–297 (2010). https://doi.org/10.1007/s11831-010-9045-2
+- Dadvand, P., Rossi, R., Gil, M., Martorell, X., Cotela, J., Juanpere, E., Idelsohn, S., Oñate, E. (2013). Migration of a generic multi-physics framework to HPC environments. Computers & Fluids. 80. 301–309. 10.1016/j.compfluid.2012.02.004.
+- Mataix Ferrándiz, V., Bucher, P., Rossi, R., Cotela, J., Carbonell, J. M., Zorrilla, R., … Tosi, R. (2020, May 29). KratosMultiphysics (Version 8.0). Zenodo. https://doi.org/10.5281/zenodo.3234644
```
Comment (philbucher) [inline README.md:102]: this is determined by the number of commits right?

Just clarifying such that it is fair for everyone

**Comment by philbucher** [comment]
looks good, thx @riccardotosi

File: README.md (hunk 1 above)
Comment (riccardotosi) [reply README.md:102]: I guess so, but I cannot confirm. @roigcarlo created the release in zenodo and maybe he knows more.

Comment (loumalouomega) [reply README.md:102]: I agree :) ...

**Comment by marandra** [comment]
I see that Zenodo provides a badge with the DOI. Maybe add it to the README too?

[DOI](https://doi.org/10.5281/zenodo.3870709)

**Comment by riccardotosi** [comment]
> I see that Zenodo provides a badge with the DOI. Maybe add it to the README too?
> 
> [DOI](https://doi.org/10.5281/zenodo.3870709)

What do you mean? A DOI is present. Can you propose the change?

File: README.md [hunk 2]
```diff
@@ -11,6 +11,9 @@
 [CI-image]: https://github.com/KratosMultiphysics/Kratos/workflows/CI/badge.svg
 [CI-link]: https://github.com/KratosMultiphysics/Kratos/actions?query=workflow%3ACI
 
+[DOI-image]: https://zenodo.org/badge/DOI/10.5281/zenodo.3870709.svg
+[DOI]: https://doi.org/10.5281/zenodo.3870709
```
Comment (riccardotosi) [inline README.md]:
```suggestion
[DOI]: https://doi.org/10.5281/zenodo.3234644
```

File: README.md [hunk 3]
```diff
@@ -11,6 +11,9 @@
 [CI-image]: https://github.com/KratosMultiphysics/Kratos/workflows/CI/badge.svg
 [CI-link]: https://github.com/KratosMultiphysics/Kratos/actions?query=workflow%3ACI
 
+[DOI-image]: https://zenodo.org/badge/DOI/10.5281/zenodo.3870709.svg
```
Comment (riccardotosi) [inline README.md]:
```suggestion
[DOI-image]: https://zenodo.org/badge/DOI/10.5281/zenodo.3234644.svg

```

File: README.md (hunk 1 above)
Comment (roigcarlo) [reply README.md:102]: This list is automatically generated from  the public info available in github and as you say is ordered by the number of commits. This ordering is not enforced and I can change it any time you want. I can also add a ORCID to your profile etc...  / Change affiliation center(s) / change your name etc...

If you have any request please tell me. Also changing that will not change the DOI number whatsoever, so please don't wait for a new release. (send me the requests to croig@cimne.upc.edu)

Also worth mention that the DOI citation for the Kratos repo references only the last release of that software in zenodo page, and no other derived works /versions.

**Review by loumalouomega — APPROVED** [review]

**Comment by riccardotosi** [comment]
I will merge tomorrow afternoon, so everyone has the time to see this PR open and @roigcarlo comment. Let me know if you want to wait more.

**Pull request merged**